pkg/output/output.go

```diff
@@ -210,6 +216,32 @@ func (b *Handler) writeSIGRetestBadge(name, filePath string, data types.RunningA
 	return badge.Render(name, badgeString, color, f)
 }
 
+func (b *Handler) writeJobFailureBadges(data types.RunningAverageDataItem, levels *Levels) error {
+	failedJobLeaders := data.FailedJobLeaderBoard
+	basePath, err := b.initializeSourcePath()
+	if err != nil {
+		return err
+	}
+	for i, job := range failedJobLeaders {
+		if i < 10 {
```

Comment:
Why not `for i, job := range failedJobLeaders[:10]` with assumption you have 10 or more all the time... if anything please do early return/break `if i>=10{ break/return}`

Reply:
Updated with `for i, job := range failedJobLeaders[:10]`

Reply:
Going to go with the early break to satisfy the tests -  `runtime error: slice bounds out of range [:10] with capacity 4`

Reply:
What would happen if there are less than 10 failed  jobs? Can we prepare for this happy occasion by generating green "None 0/0" badges (or no badges at all)?